packages/dynamodb-data-mapper/src/DataMapper.ts

```diff
@@ -280,6 +281,7 @@ export class DataMapper {
             streamViewType = 'NONE',
             writeCapacityUnits,
             indexOptions = {},
+            billingMode = 'PROVISIONED',
```

Comment:
Since `BillingMode` will be defaulted to `'Provisioned'`, I don't think a default value should be injected here. Let the `undefined` pass forward to the `CreateTableRequest` defined on line 293.